NOTICE

```diff
@@ -223,11 +223,16 @@ It is under the MIT license with an optional exception:
     without including the above copyright and permission notices.
 
 For src/types/geohash.*, src/common/rand.*, src/common/sha1.*, src/storage/scripting.* and
-some functions in src/common/util.cc are modified from Redis.
+some functions in src/common/*_util.* are modified from Redis.
 
 We have added the Redis license notice in each file header, can also see bundled license file
 https://github.com/apache/incubator-kvrocks/blob/unstable/licenses/LICENSE.redis for details.
 
+For src/common/rocksdb_crc32c.* and src/storage/batch_debugger.* are modified from RocksDB.
+
+We have added the RocksDB license notice in each file header, can also see bundled license file
+https://github.com/apache/incubator-kvrocks/blob/unstable/licenses/LICENSE.rocksdb for details.
```

Comment:
IIRC We don't have to duplicate ALv2 since the content is the same. But we do need to declare that we choose the ALv2 for RocksDB dependency since it's dual licensed by ALv2 and GPLv2.

Reply:
done